tests/test_client.py

```diff
@@ -2,40 +2,29 @@
 from dotenv import load_dotenv, find_dotenv
 from todo_app import app
 import os
-import requests
+import mongomock
+from pymongo import MongoClient
 
 from datetime import date
-from todo_app.data.trello_items import get_items
 
 @pytest.fixture
 def client():
     file_path = find_dotenv('.env.test')
     load_dotenv(file_path, override=True)
-    test_app = app.create_app()
-    
-    with test_app.test_client() as client:
-        yield client
 
-def test_index_page(monkeypatch, client):
-    monkeypatch.setattr(requests, 'get', get_items)
-    response = client.get('/')
-    assert response.status_code == 200
-    assert 'Test card' in response.data.decode() 
```

Comment:
I've tried adding this back in but I'm getting an error that I'm struggling to debug :'( 

`pymongo.errors.ServerSelectionTimeoutError: fakemongo.com:27017: [Errno 11001] getaddrinfo failed, Timeout: 30s, Topology Description: <TopologyDescription id: 63499102a8ca2072f1f48ca1, topology_type: Unknown, servers: [<ServerDescription ('fakemongo.com', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('fakemongo.com:27017: [Errno 11001] getaddrinfo failed')>]>                                                                            meout: 30s, Topology Description: <TopologyDescriptio
                                                                                                                     type: Unknown, rtt: None, error=AutoReconnect('fakemo
.venv\lib\site-packages\pymongo\topology.py:238: ServerSelectionTimeoutError`

Reply:
Oh, this is an annoying one.. in mongo_db_client.py do `import pymongo` and them `pymongo.MongoClient` rather than `from pymongo import MongoClient`. mongomock can't patch the second one

Reply:
Ah thanks, that worked! I'm not sure how to re-request approval on github anymore - was there anything else I missed?

Reply:
That was the only thing, can you resubmit on Aptem so we can approve it there?

Feel free to merge in the meantime